src/controllers/transactionController.ts

```
@@ -31,6 +32,7 @@ export async function listMyTransactions(
       throw new AppError("User-scoped API key required", 401, "UNAUTHORIZED");
     }
 
+    const query = listTransactionsQuerySchema.safeParse(req.query as Record<string, unknown>);
```

ESLint

[error] Replace `req.query·as·Record<string,·unknown>` with `⏎······req.query·as·Record<string,·unknown>,⏎····`

(prettier/prettier)